src/schema-gen/InspectorXSLT/testing/current/computer_inspector.xsl

```diff
@@ -0,0 +1,2360 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:transform xmlns="http://www.w3.org/1999/xhtml"
+                xmlns:mx="http://csrc.nist.gov/ns/csd/metaschema-xslt"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                version="3.0"
+                xpath-default-namespace="http://example.com/ns/computer"
+                exclude-result-prefixes="#all"><!-- Generated 2024-01-03T11:37:11.730831-05:00 -->
```

Comment:
Minor: I have a difference in philosophy, but for "compiled" (or rather transpiled assets), I do not like adding them and committing them to the tree, especially when they have time-based/freshness-based timestamps like on this line. It adds git diffs and unintentional noise to commits, so I usually exclude those and publish them otherwise out of band for that reason.

This pattern causes over time a large history and many deltas that impacted the OSCAL project in particular and lead to the massive architecture change in website development and operation last year (since GitHub Actions uses the repos). This pattern is in a smaller repo far from that, but caveat emptor. You are the maintainer and can do that, but I always avoid that in my more recent work.

Reply:
I agree with this, but I haven't yet managed to codify it or enforce it systematically. Things change too quickly.

In this case, there are unexplored issues relating to potential tests that use (depend on) these being in place and current ... so tbd ...

Reply:
(I'm also going to come back to this in the WIP branch.)